src/pages/wallet/Activity.vue

```diff
@@ -115,9 +115,13 @@ const filteredTransaction = computed(() => {
     <ActivityItem :activity="tx" />
   </div>
   <Teleport to="#rightPanel">
-    <div class="hidden sm:flex ml-auto w-fit">
-      <SelectField v-model="actionType" class="mr-4" :items="actionTypes" />
-      <SelectField v-model="period" :items="periods" />
+    <div class="flex ml-auto w-fit lt-sm:flex-col">
```

Comment:
Have both the filters in 1 row instead of 1 row each.

Reply:
![image](https://user-images.githubusercontent.com/94340415/155264154-300ba37b-4b73-47f9-9ea3-bb40e37bae5a.png)